docs/api-status.md

```diff
@@ -12,10 +12,19 @@ No Preview/Deprecated APIs found. All APIs are considered stable.
 
 ## Package: rados
 
-No Preview/Deprecated APIs found. All APIs are considered stable.
-
```

Comment:
You can take out this change(unrelated) as it might be a bug with the `make api-update` process.

Reply:
Thank you for the review @anoopcs9. Made the change